rust/agama-dbus-server/src/network/model.rs

```diff
@@ -679,6 +682,20 @@ impl From<&IpRoute> for HashMap<&str, Value<'_>> {
     }
 }
 
+#[derive(Debug, Default, PartialEq, Clone)]
+pub enum VlanProtocol {
+    #[default]
+    Ieee802_1Q,
+    Ieee802Ad,
```

Comment:
Why not 
```suggestion
    Ieee802_1Ad,
```
in this case?

Reply:
Indeed, that's more precise.

Reply:
changed to `IEEE802_1Q` and `IEEE802_1ad`